src/mediapipe_internal/mediapipegraphexecutor.cpp

```diff
@@ -448,14 +449,16 @@ static Status deserializeTensor(const std::string& requestedName, const KFSReque
 MediapipeGraphExecutor::MediapipeGraphExecutor(const std::string& name, const std::string& version, const ::mediapipe::CalculatorGraphConfig& config,
     stream_types_mapping_t inputTypes,
     stream_types_mapping_t outputTypes,
-    std::vector<std::string> inputNames, std::vector<std::string> outputNames) :
+    std::vector<std::string> inputNames, std::vector<std::string> outputNames,
+    std::map<std::string, std::shared_ptr<PythonNodeResource>>& pythonNodeResources) :
```

Comment:
You probably want to pass **const** reference because you do not modify it anyway